src/LensPlatformClient.ts

```diff
@@ -154,9 +154,9 @@ class LensPlatformClient {
    */
   get fetch() {
     const getToken = this.getToken.bind(this);
-    const defaultHeaders = this.defaultHeaders;
-    const httpAdapter = this.httpAdapter;
-    const logger = this.logger;
+    const { defaultHeaders } = this;
```

Comment:
you could do `const { defaultHeaders, httpAdapter, logger } = this;`

Reply:
True.. I think this was eslint's change